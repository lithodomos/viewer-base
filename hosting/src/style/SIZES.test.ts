import {
  SCREEN_WIDTH_BREAK_EXTRA_LARGE,
  SCREEN_WIDTH_BREAK_LARGE,
  SCREEN_WIDTH_BREAK_MEDIUM,
  SCREEN_WIDTH_BREAK_SMALL,
  SCREEN_WIDTH_BREAK_EXTRA_SMALL,
  SCREEN_WIDTH_MAX,
  HEADER_HEIGHT,
  HEADER_WIDTH,
  HEADER_BUTTON_SIZE,
  ICON_BUTTON_SIZE,
} from './SIZES';

test('it matches the snapshot', () => {
  expect(SCREEN_WIDTH_BREAK_EXTRA_LARGE).toMatchSnapshot();
  expect(SCREEN_WIDTH_BREAK_LARGE).toMatchSnapshot();
  expect(SCREEN_WIDTH_BREAK_MEDIUM).toMatchSnapshot();
  expect(SCREEN_WIDTH_BREAK_SMALL).toMatchSnapshot();
  expect(SCREEN_WIDTH_BREAK_EXTRA_SMALL).toMatchSnapshot();
  expect(SCREEN_WIDTH_MAX).toMatchSnapshot();
  expect(HEADER_HEIGHT).toMatchSnapshot();
  expect(HEADER_WIDTH).toMatchSnapshot();
  expect(HEADER_BUTTON_SIZE).toMatchSnapshot();
  expect(ICON_BUTTON_SIZE).toMatchSnapshot();
});
