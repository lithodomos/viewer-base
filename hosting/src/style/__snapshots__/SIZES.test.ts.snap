// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches the snapshot 1`] = `1100`;

exports[`it matches the snapshot 2`] = `960`;

exports[`it matches the snapshot 3`] = `800`;

exports[`it matches the snapshot 4`] = `676`;

exports[`it matches the snapshot 5`] = `500`;

exports[`it matches the snapshot 6`] = `1200`;

exports[`it matches the snapshot 7`] = `50`;

exports[`it matches the snapshot 8`] = `240`;

exports[`it matches the snapshot 9`] = `16`;

exports[`it matches the snapshot 10`] = `20`;
